gcs-avro-tools.rb

```diff
@@ -0,0 +1,22 @@
+class GcsAvroTools < Formula
+  desc "GCS compatible Apache Avro Tools"
+  homepage "https://github.com/spotify/gcs-tools"
+  url "https://github.com/spotify/gcs-tools/releases/download/v0.1.1/avro-tools-1.8.1.jar"
+  sha256 "382f2bfd756b190e4bad5038807705ae66dda132e17a0ac88a0069f246e720d0"
+  version "0.1.1"
+
+  conflicts_with "avro-tools", :because => "provides the same binaries/links."
```

Comment:
"Abort" on conflict. Looks like this:

```
➜ brew install gcs-avro-tools
==> Installing gcs-avro-tools from spotify/public
Error: Cannot install spotify/public/gcs-avro-tools because conflicting formulae are installed.

  avro-tools: because provides the same binaries/links.

Please `brew unlink avro-tools` before continuing.

Unlinking removes a formula's symlinks from /usr/local. You can
link the formula again after the install finishes. You can --force this
install, but the build may fail or cause obscure side-effects in the
resulting software.
```

So says exactly what to do to fix it. Good enough. Also remove of `#{bin}/avro-tools` did not work as a brute force idea (if avro tools are already installed).